app/plugin.go

```diff
@@ -255,7 +256,20 @@ func (a *App) SyncPlugins() *model.AppError {
 			continue
 		}
 		defer reader.Close()
-
+		var sigReader filesstore.ReadCloseSeeker
+		sigReader, appErr = a.FileReader(fmt.Sprintf("%s.sig", path))
+		if appErr != nil {
```

Comment:
Is there a way to distinguish a missing file from another error, and log a more detailed message distinguishing these cases?